Taskfile.yml

```diff
@@ -97,6 +105,11 @@ tasks:
           [ "$(which cargo-insta)" ] || echo "🔴 Can't find a binary that cargo just installed. Is the cargo bin path (generally at `~/.cargo/bin`) on the \$PATH?"
         silent: true
 
+  install-cargo-tools-admin:
+    desc: Install cargo tools for PRQL development & admin.
+    cmds:
+      - cargo install --locked {{.cargo_crates_admin | splitLines | join " " }}
```

Comment:
Why don't use bintall?

Reply:
We only use `binstall` for CI, because it would write hundreds of `429` messages to the terminal when running locally.

Maybe that's improved? Though it installing the wrong version of `cargo admonish` doesn't give me confidence we should expand our use of it...

Reply:
> because it would write hundreds of 429 messages to the terminal when running locally.

I don't understand exactly what happens, but wouldn't it be solved by simply suppressing the output to the terminal?

Reply:
But then there's no output for 10 minutes?

Reply:
Ok, it certainly may not be a good idea to suppress everything.
But given the frequency with which this command is executed, I'm not sure if the large amount of output to the console is a compelling reason not to use binstall.
At least for me, I prefer a lot of logging to spending dozens of minutes compiling.